components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/APGARController.java

```diff
@@ -180,6 +182,9 @@ private boolean hasAnySelected(Collection<String> selectedFieldNames)
                     if (NumberUtils.isDigits(value)) {
                         parsed.put(propertyName, Integer.valueOf(value));
                     }
+                    if (StringUtils.isEmpty(value)) {
+                        parsed.put(propertyName, null);
```

Comment:
as discussed, we probably want a separate issue to consistently represent missing data with empty strings or nulls, and not a random mix of 0-s, null-s and empty strings like we do now